crates/rolldown_testing_config/src/lib.rs

```diff
@@ -1,13 +1,16 @@
 use std::fmt::Display;
 
-use rolldown_common::{BundlerOptions, OutputFormat};
+use rolldown_common::{BundlerOptions, OutputExports, OutputFormat};
 use schemars::JsonSchema;
 use serde::Deserialize;
 
 #[derive(Deserialize, JsonSchema, Default)]
 #[serde(rename_all = "camelCase", deny_unknown_fields)]
 pub struct ConfigVariant {
   pub format: Option<OutputFormat>,
+  pub extend: Option<bool>,
```

Comment:
These new fields should be added in

https://github.com/rolldown/rolldown/blob/d5e30898d9a34413afa524212aee476c1f9e7ca5/crates/rolldown_testing_config/src/lib.rs#L38

, so we could visualize them easily in snapshot.